### DISCUSSION TITLE
Which mouse events work for sliders?

### DISCUSSION THREAD

**Discussion by RunDevelopment** [chakra-ui/chakra-ui#7087] [Q&A]
Hi!

I recently needed the mouse down even of a slider, but found that the `onMouseDown` event handler function is never called. I find this strange because other mouse events like `onDoubleClick`, `onMouseEnter`, and `onMouseLeave` do work.

[The docs](https://chakra-ui.com/docs/components/slider/props) say that "the Slider component wraps all its children in the [Box](https://chakra-ui.com/docs/layout/box) component, so you can pass all Box props [...]", so I would expect all Box props to work.

So, which events are and are not supported for sliders?

**Comment by 1pxone** [comment]
Hey:) Any updates here? Should comunity fix this? (seems like it would be easy journey)

**Comment by mayataso** [comment]
Hi.
I too need the `onMouseDown` event for the slider and would like to be able to detect it.

At least,
> [The docs](https://chakra-ui.com/docs/components/slider/props) say that "the Slider component wraps all its children in the [Box](https://chakra-ui.com/docs/layout/box) component, so you can pass all Box props [...]", so I would expect all Box props to work.

I think this comment is correct.

Is it possible?
Or, if you have an alternative hack, please let me know.

Thanks.
